### PR TITLE
Admin dashboard orders

### PR DESCRIPTION
**Pull request by heraimenka**
Worked on the dashboard page for admins

Implemented:
- Searching
- Sorting by price
- Editing orders
- Minor changes in styling

### CODE REVIEW

File: src/components/pages/Dashboard/index.js [hunk 1]
```diff
@@ -20,10 +24,10 @@ function Dashboard(props) {
   const orders = props.orders;
   const { authState } = useOktaAuth();
 
-  // load orders on first render
+  // load orders
   useEffect(() => {
     props.ordersGet(authState);
-  }, []);
+  }, [props.orders]);
```
Comment (wSedlacek) [inline src/components/pages/Dashboard/index.js]: This would cause an infinite loop

File: src/state/actions/ordersActions.js [hunk 2]
```diff
@@ -76,9 +79,22 @@ export const orderDelete = (authState, id) => dispatch => {
...
-export const orderEdit = editingOrder => dispatch => {
+export const orderEdit = (authState, id, updatedData) => dispatch => {
   dispatch({
-    type: ORDER_EDIT_SUCCESS,
-    payload: editingOrder,
+    type: ORDER_EDIT_LOADING,
   });
+
+  editOrderData(authState, id, updatedData)
+    .then(payload => {
```
Comment (wSedlacek) [inline src/state/actions/ordersActions.js]: here this should be `.then(({ order })=> {`

File: src/state/actions/ordersActions.js [hunk 3]
```diff
@@ -76,9 +79,22 @@ export const orderDelete = (authState, id) => dispatch => {
...
+      dispatch({
+        type: ORDER_EDIT_SUCCESS,
+        payload: id,
```
Comment (wSedlacek) [inline src/state/actions/ordersActions.js]: Here the payload should probably be `paylaod: order`

Comment (wSedlacek) [reply src/state/actions/ordersActions.js]: We seem to be missing the reducer for this.

Comment (heraimenka) [reply src/state/actions/ordersActions.js]: Added reducers

File: src/components/pages/Dashboard/index.js (hunk 1 above)
Comment (heraimenka) [reply src/components/pages/Dashboard/index.js]: Changed it back to [ ]

File: src/state/actions/ordersActions.js (hunk 2 above)
Comment (heraimenka) [reply src/state/actions/ordersActions.js]: Changed

**Review by wSedlacek — APPROVED** [review]
LGTM

**Pull request merged**